Sources/SwiftDocC/Model/Kind.swift

```diff
@@ -192,9 +192,9 @@ extension DocumentationNode.Kind {
         // Grouping
         .landingPage, .collection, .collectionGroup,
         // Conceptual
-        .root, .module, .article, .sampleCode, .technologyOverview, .volume, .chapter, .tutorial, .tutorialArticle, .onPageLandmark,
+        .root, .module, .article, .sampleCode, .volume, .chapter, .tutorial, .tutorialArticle, .onPageLandmark,
         // Containers
-        .class, .structure, .enumeration, .protocol, .technology, .extension, .dictionary, .httpRequest, .namespace,
+        .class, .structure, .enumeration, .protocol, .tutorialTableOfContents, .extension, .dictionary, .httpRequest, .namespace,
```

Comment:
After this PR is merged, DocC won't support `.technology` and `.technologyOverview ` as valid values anymore, as per the changes in commit 7070e86e80dd4fbb3f406e6a89a44ced6d616ac7. 

I think we should keep supporting them in order to avoid any backward compatibility issues.

Reply:
I added back the `.technologyOverview` in https://github.com/swiftlang/swift-docc/pull/1039/commits/6ed54e94eee2515695cfd7057adcd267474b33c2 but I don't think that adding back `.technology` would cause more issues since it make would violate the uniqueness of the kind's `id`

> A globally unique identifier for the kind

If you look closely I didn't change the identifier when I renamed "Technology" to "Tutorial Table of Contents". 

I don't think we should consider the kind's display name's a necessarily stable. For example, I feel that we should permit ourselves to change the capitalization from title case to sentence case without considering that a breaking change. Any caller should use the `id` to identify a kind, not its display name.

Reply:
This sounds good, I think having the ID remain the same is good enough. Maybe we should add a comment which says exactly this?
> Any caller should use the id to identify a kind, not its display name.